### DISCUSSION TITLE
Translated wiki pages

### DISCUSSION THREAD

**Discussion by ghost** [openstreetmap/iD#10272] [Q&A] [answered]
Hi there !

I've translated some pages into French for the wiki but on id editor, it's still in English. Do I need to update a config file or something else?

Thanks in advance.
Respectfully

**Comment by ghost** [comment]
Any news ?

**Comment by 1ec5** [comment]
Which pages are these, so that we can investigate?

**Reply by ghost** [reply]
@1ec5 

Voila !
https://wiki.openstreetmap.org/wiki/FR:Key:cash_in

**Reply by Oivo35** [reply]
It would be good to add here a screenshot of iD, what field do you want to translate.

**Reply by ghost** [reply]
@Oivo35 

It's this one
![Copie d'écran_20240609_154741](https://github.com/openstreetmap/iD/assets/116185937/5d661bc9-c98e-40ca-8f55-b193f5512b60)

**Reply by 1ec5** [reply, marked as answer]
This description comes from the [data item](https://wiki.openstreetmap.org/wiki/FR:Data_items) for `cash_in=*`. I’ve [updated the data item](https://wiki.openstreetmap.org/wiki/Special:Diff/2654034/2719600) to include a French description and a link to the French article. Now it should show up in the help text:

<img src="https://github.com/openstreetmap/iD/assets/1231218/28f9ce19-b7d1-462c-a833-e94eb64bab34" width="372" alt="Précise si le DAB peut être utilisé pour déposer de l'argent">

In the future, when you create a new translation for a tagging page, you can click on “Data item” in the sidebar of the original article to access the data item. Once you tag the data item with a “[pages de documentation wiki <small>(P31)</small>](https://wiki.openstreetmap.org/wiki/Property:P31)” statement in French, “Élément de données” will also appear in the sidebar of the French article.

**Reply by ghost** [reply]
Thanks a lot ^^.
